src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,9 @@
+export const Tab = ({ tab, findActive, active }) => {
+  return (
+    <li data-cy="Tab" className={tab.id === active.id ? 'is-active' : ''}>
+      <a href={`#${tab.id}`} data-cy="TabLink" onClick={() => findActive(tab)}>
```

Comment:
Change `findActive(tab)` to `onTabSelected(tab)` to align with the updated prop name.